js/src/product-feed/review-request/review-request-notice.js

```diff
@@ -46,16 +45,9 @@ const ReviewRequestNotice = ( {
 							{ REVIEW_STATUSES[ account.status ].body }
 							{ isDisabled &&
 								createInterpolateElement(
-									sprintf(
-										// translators: %s: The date to wait until the next request review.
-										__(
-											' You can request a new review on %s. <Link>Learn more</Link>',
-											'google-listings-and-ads'
-										),
-										format(
-											'd M Y',
-											account.nextAttempt * 1000
-										)
+									__(
+										' You can request a new review approximately 7 days after a disapproval. <Link>Learn more</Link>',
```

Comment:
❓

Is it not possible to add the text 'You can request a new review ....' into the `REVIEW_STATUS`, so we have all texts in one place?

Reply:
Good suggestion, That will improve the code :) 

Added some refactoring here: https://github.com/woocommerce/google-listings-and-ads/pull/1256/commits/95371be924faa81700717273ed4f3ed6dfc38efc

About your suggestions and a few more refactors on that file. 

Let me know what do u think.